app/assets/javascripts/locale/cs_CZ/app.js

```diff
@@ -1 +1 @@
iated with a VM":[""],"No parameters were allocated to the selected hosts, can't mass assign":[""],"Updated all hosts!":["Aktualizovány všechny stroje!"],"%s Parameters updated, see below for more information":["%s Parametr aktualizován, další informace viz níže"],"No host group selected!":["Nevybrána žádná skupina hostitelů!"],"Updated hosts: changed host group":[""],"No owner selected!":["Nevybrán žádný vlastník!"],"Updated hosts: changed owner":[""],"Failed to set power state for %s.":[""],"The power state of the selected hosts will be set to %s":[""],"%{config_type} rebuild failed for host: %{host_names}.":["","","",""],"Configuration successfully rebuilt":["Nastavení úspěšně znovu vytvořena"],"Failed to redeploy %s.":[""],"The selected hosts were enabled for reboot and rebuild":[""],"The selected hosts will execute a build operation on next reboot":["Označení hostitelé vykonají operaci sestavení při příštím restartu"],"The following hosts failed the build operation: %s":["Na následujících hostitelích se nezdařila operace sestavení: %s "],"Destroyed selected hosts":["Označené stroje zlikvidovány"],"The following hosts were not deleted: %s":["Následující stroje nebyly smazány: %s"],"Updated hosts: Disassociated from VM":[""],"Hosts with errors":["Stroje s chybami"],"Active Hosts":["Aktivní stroje"],"Pending Hosts":["Čekající stroje"],"Hosts which didn't report in the last %s":[""],"Hosts with notifications disabled":["Hostitelé pro které je upozorňování vypnuté"],"invalid type: %s requested":["neplatný typ: %s vyžadováno"],"Something went wrong while changing host type - %s":[""],"No hosts were found with that id, name or query filter":["Pro takový identifikátor, název nebo dotazovací filtr nebyly nalezeni žádní hostitelé"],"No hosts selected":["Nejsou vybrány žádné stroje"],"Something went wrong while selecting hosts - %s":["Při vybírání strojů se něco pokazilo – %s"],"%s selected hosts":["vybráno %s strojů"],"The following hosts were not %{action}: %{missed_hosts}":[""],"No or invalid power state selected!":[""],"The %{proxy_type} proxy of the selected hosts was cleared":[""],"No templates found":["Nenalezeny žádné šablony"],"HTTP Proxy connection successful.":["Připojení k HTTP proxy úspěšné."],"%{user} Downloaded %{key} as pem file":["%{user} Stažen %{key} v podobě pem souboru"],"Successfully recreated":[""],"Could not recreate a new SSH key":["Nedaří se znovu vytvořit nový SSH klíč"],"Successfully delete %s":["Úspěšně smazat %s"],"Could not delete %s":["%s se nedaří smazat"],"Successfully overridden all parameters of Puppet class %s":[""],"Successfully reset all parameters of Puppet class %s to their default values":[""],"No parameters to override for Puppet class %s":[""],"Report is going to be rendered at %s and it will be delivered via e-mail once ready.":[""],"Report is being rendered, it will be delivered via e-mail.":["Výkaz je vykreslován, bude doručen prostřednictvím e-mailu."],"Could not generate the report, check the form for er
```

Comment:
this one is I believe generated, please drop it from this PR

Reply:
Dropped from commit and force-pushed.